influencers_app/enums.py

```diff
@@ -0,0 +1,19 @@
+from django.utils.translation import gettext_lazy as _
+from enum import IntEnum
+
+
+class Progress(IntEnum):
+
+    REVIEW_DONE = 1, _('Review done')
```

Comment:
Enum class objects should be declared like this:
class ProgressType(IntEnum):
    REVIEW_DONE = 1
...

You can also set auto() to all of yours enum objects instead of using 1, 2, 3... numbering. The result will be the same.
If you need to add some string mapping to your enum class you should create a dict with corresponding mapping:
PROGRESS_TYPES_MAPPING = {ProgressType.REVIEW_DONE: "Review Done"}